fe/fe-core/src/main/java/com/starrocks/sql/optimizer/Optimizer.java

```diff
@@ -333,6 +333,16 @@ private void collectAllScanOperators(Memo memo, TaskContext rootTaskContext) {
         rootTaskContext.setAllScanOperators(Collections.unmodifiableList(list));
     }
 
+    private void rewriteSubquery(Memo memo, TaskContext rootTaskContext) {
+        do {
```

Comment:
comment why need to rewrite PUSH_DOWN_SUBQUERY rule exhaustive

Reply:
when multi subquery. like `where v1 < (subquery) and v2 > (subquery)`.

like 
```
Apply
    |
Apply
    |
Project
```

I want to get
```
Project
    |
Apply
    |
Apply
```

But it's hard on our framework

Reply:
what I mean is comment in code.